starter-code/index.html

```diff
@@ -3,40 +3,90 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link href="./css/main.css" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- nav bar -->
 
-What’s on Spotify?
+<div class="nav-bar">
+  <div class="logo"><img src="./images/spotify-logo.png" alt="logo"></div>
+<div class="links">
+  <ul>
+    <li>Premium</li>
+    <li>Discover</li> 
+    <li>Help</li> 
+    <li>Download</li>
+  </ul>
+</div>
+</div>
+<div class="clear"></div>
+<!-- music for everyone -->
 
-Millions of Songs
-There are millions of
-songs on Spotify
+<div class="everyone">
```

Comment:
I do not like your choice for this div name "everyone"... ( see my comment below about the reason why )